html/elements/script.json

```diff
@@ -507,26 +519,38 @@
                 "edge_mobile": {
                   "version_added": "16"
                 },
-                "firefox": {
-                  "version_added": true,
-                  "flags": [
-                    {
-                      "type": "preference",
-                      "name": "dom.moduleScripts.enabled",
-                      "value_to_set": "true"
-                    }
-                  ]
-                },
-                "firefox_android": {
-                  "version_added": true,
-                  "flags": [
-                    {
-                      "type": "preference",
-                      "name": "dom.moduleScripts.enabled",
-                      "value_to_set": "true"
-                    }
-                  ]
-                },
+                "firefox": [
+                  {
+                    "version_added": "60"
+                  },
+                  {
+                    "version_added": true,
```

Comment:
Looks like this was added in 54. See https://bugzilla.mozilla.org/show_bug.cgi?id=1330657